packages/web/docs/src/components/landing-page.tsx

```diff
@@ -29,9 +29,13 @@ const useIsomorphicLayoutEffect = typeof window !== 'undefined' ? useLayoutEffec
 
 export function IndexPage(): ReactElement {
   useIsomorphicLayoutEffect(() => {
```

Comment:
I don't get it, why on server we use `useEffect` and on client `useLayoutEffect`? `useEffect` is never called on server

Reply:
We want to keep the same number of hooks so it doesn't count as calling them conditionally.

I really only want this useLayoutEffect, but it warns in server render.

https://github.com/streamich/react-use/blob/master/docs/useIsomorphicLayoutEffect.md

> ⚠️ Warning: useLayoutEffect does nothing on the server, because its effect cannot be encoded into the server renderer’s output format. This will lead to a mismatch between the initial, non-hydrated UI and the intended UI. To avoid this, useLayoutEffect should only be used in components that render exclusively on the client. See https://fb.me/react-uselayouteffect-ssr for common fixes.